### DISCUSSION TITLE
auto labeling guide

### DISCUSSION THREAD

**Discussion by kostaspic** [doccano/doccano#1588] [Q&A]
Hello,
I would like to ask if there is a tutorial / guide on how to use / set up the auto labeing tool via the web interface?
Thank you in advance!
 
![image](https://user-images.githubusercontent.com/79839747/144601449-ccb5aeaa-043a-4f2b-8ae7-ea53a9aeaf4e.png)

**Comment by Hironsan** [comment]
Added a document:
- [Setup Auto Labeling](https://github.com/doccano/doccano/blob/master/docs/advanced/auto-labelling-config.md)

**Reply by cliuxinxin** [reply]
hi, I found if I set up my own server . The doc didn't help me . sample text how to get in to body. 

<img width="781" alt="image" src="https://user-images.githubusercontent.com/6162238/160098663-4c226075-178a-4403-a88b-206e59e7b1cf.png">

<img width="765" alt="image" src="https://user-images.githubusercontent.com/6162238/160098908-f5870f29-aca4-4a48-afa3-cee32ed1268f.png">

**Comment by cliuxinxin** [comment]
I solved all my problems . And it worked. this is how I set it.

<img width="896" alt="image" src="https://user-images.githubusercontent.com/6162238/160354477-89cb4dc2-07fc-43af-b7cb-1a6f6214c9c6.png">
